catalyst/contrib/models/cv/encoders/resnet.py

```diff
@@ -22,7 +22,7 @@ def __init__(
         state_dict: Union[dict, str, Path] = None,
     ):
         """
-        Specifies an encoder for classification network
+        Specifies an encoders for classification network
```

Comment:
```suggestion
        Specifies encoders for classification network
```

That's grammatically correct.